test/integration/start_stop_delete_test.go

```diff
@@ -174,7 +174,7 @@ func TestStartStop(t *testing.T) {
 	})
 }
 
-func TestStartStopWithPreload(t *testing.T) {
+func TestPreloadStartStop(t *testing.T) {
```

Comment:
how about renaming it to TestPreload, how about moving it to its own file , this file is getting too big.
preload_test.go ?